src/Result.js

```diff
@@ -114,45 +168,19 @@ class Result extends Component {
             </Col>
             <Col sm={{ span: 6, offset: 1 }}>
               <div className="font-medium">About</div>
-              <div>More about this experience coming soon!</div>
+              <div>{this.state.description}</div>
```

Comment:
I think we may want to considering adding a conditional here so that if a description can't be scraped by Bing it displays the original message.